smithy-model/src/test/java/software/amazon/smithy/model/transform/DowngradeToV1Test.java

```diff
@@ -122,9 +122,16 @@ public void removesUnnecessaryDefaults() {
                 + "namespace smithy.example\n"
                 + "@default(10)\n"
                 + "integer MyInteger\n"
+                + "@default(0)\n"
+                + "integer ZeroInteger\n"
+                + "integer BoxedInteger\n"
                 + "structure Struct {\n"
                 + "  @default(10)\n"
                 + "  foo: MyInteger\n"
+                + "  defaultZero: ZeroInteger = 0\n"
+                + "  boxedZero: ZeroInteger = null\n"
+                + "  boxedInteger: BoxedInteger = 0\n"
```

Comment:
`boxedInteger` having a default 0 and `boxedMemberInteger` being boxed seems confusing.
maybe?
```
boxedIntegerWithMemberDefault: BoxedInteger = 0
boxedInteger: BoxedInteger = null
```

Reply:
Will rename the added members.